resources/facts.txt

```diff
@@ -68,3 +68,54 @@ Several ships have wrecked on Disappointment Island.
 The Eiffel Tower is repainted every seven years.
 In 1992, a shipping crate containing 29,000 rubber ducks, beavers, turtles, and frogs fell overboard. They washed up around the world for the next 20 years, and have been used to track ocean currents.
 A group of lemurs is called a conspiracy.
+Winston Churchill was born in a ladies` room during a dance.
+The top 3 health-related searches on the Internet are (in this order): Depression, Allergies, & Cancer.
+A snail can sleep for three years.
+Barbie`s full name is Barbara Millicent Roberts.
+A shark is the only fish that can blink with both eyes.
+There are 269 steps to the top of the Leaning Tower of Pisa
+The Ramses brand condom is named after the great pharoh Ramses II who fathered over 160 children.
+The Boston University Bridge (on Commonwealth Avenue, Boston, Massachusetts) is the only place in the world where a boat can sail under a train driving under a car driving under an airplane.
+All polar bears are left-handed.
+The state of Florida is bigger than England!
+The San Francisco Cable cars are the only mobile National Monuments.
+The U.S. military operates 234 golf courses
+John Lennon`s first girlfriend was named Thelma Pickles.
+A raisin dropped in a glass of fresh champagne will bounce up and down continually from the bottom of the glass to the top.
+the bird is the word
+If Texas were a country, its GNP would be the fifth largest of any country in the world.
+Sex burns 360 calories per hour.
+Conception occurs most in the month of December.
+400-quarter pounders can be made from 1 cow.
+There are no clocks in Las Vegas gambling casinos.
+Porcupines can float in water.
+The average woman consumes 6 lbs of lipstick in her lifetime.
+100% of lottery winners do gain weight
+In space, astronauts are unable to cry, because there is no gravity and the tears won`t flow.
+If one places a tiny amount of liquor on a scorpion, it will instantly go mad and sting itself to death.
+In Vermont, the ratio of cows to people is 10:1
+The site with the highest number of women visitors between the age of 35 and 44 years old: Alka-Seltzer.com
+All 50 states are listed across the top of the Lincoln Memorial on the back of the $5 bill.
+1 in every 4 Americans has appeared someway or another on television.
+Thirty-five percent of people who use personal ads for dating are already married.
+This common everyday occurrence composed of 59% nitrogen, 21% hydrogen, and 9% dioxide is called a `fart`.
+The airplane Buddy Holly died in was the "American Pie." (Thus the name of the Don McLean song.)
+All polar bears are left-handed.
+The word "samba" means "to rub navels together."
+`Obsession` is the most popular boat name.
+The site with the highest number of women visitors between the age of 35 and 44 years old: Alka-Seltzer.com
+Women blink nearly twice as much as men.
+The only real person to ever to appear on a pez dispenser was Betsy Ross.
+There`s a systematic lull in conversation every 7 minutes.
```

Comment:
Lots of places claiming this but citing no sources; seems false, how would one even measure that